From 267dc282f52ca2ee06ff437b5f90325bd255d624 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:08:07 -0400
Subject: [usb] qcserial: fix multiline comment coding style

Message-id: <1410887463-20674-416-git-send-email-dzickus@redhat.com>
Patchwork-id: 95261
O-Subject: [RHEL7 PATCH 415/591] usb: qcserial: fix multiline comment coding style
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit ce1b066136a30079c4e6e81e015ad9bc2180d46f
Author: Bjorn Mork <bjorn@mork.no>
Date:   Sun Apr 27 16:47:43 2014 +0200

    usb: qcserial: fix multiline comment coding style

    Use a consistent style for all multiline comments.

    Signed-off-by: Bjorn Mork <bjorn@mork.no>
    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/qcserial.c b/drivers/usb/serial/qcserial.c
index df54405..202888f 100644
--- a/drivers/usb/serial/qcserial.c
+++ b/drivers/usb/serial/qcserial.c
@@ -222,12 +222,14 @@ static int qcprobe(struct usb_serial *serial, const struct usb_device_id *id)
  /* default to enabling interface */
  altsetting = 0;
 
- /* Composite mode; don't bind to the QMI/net interface as that
+ /*
+  * Composite mode; don't bind to the QMI/net interface as that
   * gets handled by other drivers.
   */
 
  if (is_gobi1k) {
-  /* Gobi 1K USB layout:
+  /*
+   * Gobi 1K USB layout:
    * 0: DM/DIAG (use libqcdm from ModemManager for communication)
    * 1: serial port (doesn't respond)
    * 2: AT-capable modem port
@@ -241,7 +243,8 @@ static int qcprobe(struct usb_serial *serial, const struct usb_device_id *id)
   else
    altsetting = -1;
  } else {
-  /* Gobi 2K+ USB layout:
+  /*
+   * Gobi 2K+ USB layout:
    * 0: QMI/net
    * 1: DM/DIAG (use libqcdm from ModemManager for communication)
    * 2: AT-capable modem port
-- 
1.7.1